themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -134,13 +167,19 @@
         {block name='form_field_item_password'}
           <div class="input-group js-parent-focus">
             <input
-              class="form-control js-child-focus js-visible-password"
-              name="{$field.name}"
-              title="{l s='At least 5 characters long' d='Shop.Forms.Help'}"
-              type="password"
-              value=""
-              pattern=".{literal}{{/literal}5,{literal}}{/literal}"
-              {if $field.required}required{/if}
+                {if empty($field.attributes)}
+                      class="form-control js-child-focus js-visible-password"
+                      name="{$field.name}"
+                      title="{l s='At least 5 characters long' d='Shop.Forms.Help'}"
+                      type="password"
+                      value=""
+                      pattern=".{literal}{{/literal}5,{literal}}{/literal}"
+                      {if $field.required}required{/if}
+                {else}
+                    {foreach from=$field.attributes item="value" key="attribute"}
+                        {$attribute} = "{$value}"   
+                    {/foreach}
+                {/if}
```

Comment:
It kinda force to add every attributes everytime you wants to add a new attribute, can't we just add some attributes and keep some ? Like the classes, we doesnt want to rewrite classes everytime we wants to add an attribute, maybe the name attribute, title, pattern too I don't know... what do you think ?

Reply:
Oh look like we are thinking about the same think @NeOMakinG https://github.com/PrestaShop/PrestaShop/pull/16485/files#r555817942 :sweat_smile:

Reply:
@PierreRambaud  J'étais trop jeune quand j'avais fait ce PR mdrrr